src/components/rangeslider/range_attributes.js

```diff
@@ -0,0 +1,40 @@
+/**
+* Copyright 2012-2018, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = {
+    rangemode: {
+        valType: 'enumerated',
+        values: ['auto', 'fixed', 'match'],
+        dflt: 'auto',
+        role: 'style',
+        editType: 'calc',
+        description: [
+            'Determines whether or not the range of this axis in',
+            'the rangeslider use the same value than in the main plot',
+            'when zooming in/out.',
+            'If *auto*, the autorange will be used.',
+            'If *fixed*, the `range` is used.',
+            'If *match*, the current range is used.'
+        ].join(' ')
+    },
+    range: {
+        valType: 'info_array',
+        role: 'style',
+        items: [
+            {valType: 'any', editType: 'calc'},
+            {valType: 'any', editType: 'calc'}
+        ],
+        editType: 'calc',
+        description: [
+            'Sets the range of this axis for the rangeslider.'
+        ].join(' ')
+    },
+    editType: 'calc'
+};
```

Comment:
@etpinard help me out here - what's the right way to get these attributes into the schema? Just attach them as `rangeslider.attributes.yaxis`? How then do we document that `yaxis` is a stand-in for "all the y axis names relevant to this x axis"?

Reply:
Oh sorry, you partially answered this [before](https://github.com/plotly/plotly.js/pull/2364#issuecomment-367424841)
> There's a meta key in the attribute declaration called `_isSubplotObj`

So we'll need a `Plotly.validate` test with eg `xaxis: {rangeslider: {yaxis2: {...}}}`, but this still doesn't answer the documentation question, how users are expected to know to use the respective y axis names here.

Reply:
I'm not really sure on how to use this metakey, neither how this validate test works.

Reply:
No worries @TomDemulierChevret , I'll pull down your branch and see what I can do :ok_hand: